scalding-dagon/src/test/scala/com/twitter/scalding/dagon/DataFlowTest.scala

```diff
@@ -1,8 +1,9 @@
 package com.twitter.scalding.dagon
 
-import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.FunSuite
+import org.scalatest.prop.PropertyChecks
+import org.scalatest.prop.GeneratorDrivenPropertyChecks._
 import org.scalacheck.{Arbitrary, Cogen, Gen}
-import org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks._
```

Comment:
can one do this? Here I replaced `import org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks._` with `import org.scalatest.prop.GeneratorDrivenPropertyChecks._` due to scalatestplus having version conflicts with exiting scalding dependencies. 

The tests seem to run but just wanted to double check